examples/style-swin-gan/styleswin_tt.ipynb

```diff
@@ -416,13 +427,35 @@
+      "File \u001b[0;32m~/mambaforge/envs/xfetusVE/lib/python3.11/site-packages/torch/nn/modules/module.py:1143\u001b[0m, in \u001b[0;36mModule.to.<locals>.convert\u001b[0;34m(t)\u001b[0m\n\u001b[1;32m   1140\u001b[0m \u001b[38;5;28;01mif\u001b[39;00m convert_to_format \u001b[38;5;129;01mis\u001b[39;00m \u001b[38;5;129;01mnot\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m \u001b[38;5;129;01mand\u001b[39;00m t\u001b[38;5;241m.\u001b[39mdim() \u001b[38;5;129;01min\u001b[39;00m (\u001b[38;5;241m4\u001b[39m, \u001b[38;5;241m5\u001b[39m):\n\u001b[1;32m   1141\u001b[0m     \u001b[38;5;28;01mreturn\u001b[39;00m t\u001b[38;5;241m.\u001b[39mto(device, dtype \u001b[38;5;28;01mif\u001b[39;00m t\u001b[38;5;241m.\u001b[39mis_floating_point() \u001b[38;5;129;01mor\u001b[39;00m t\u001b[38;5;241m.\u001b[39mis_complex() \u001b[38;5;28;01melse\u001b[39;00m \u001b[38;5;28;01mNone\u001b[39;00m,\n\u001b[1;32m   1142\u001b[0m                 non_blocking, memory_format\u001b[38;5;241m=\u001b[39mconvert_to_format)\n\u001b[0;32m-> 1143\u001b[0m \u001b[38;5;28;01mreturn\u001b[39;00m \u001b[43mt\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mto\u001b[49m\u001b[43m(\u001b[49m\u001b[43mdevice\u001b[49m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mdtype\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mif\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mt\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mis_floating_point\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;129;43;01mor\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[43mt\u001b[49m\u001b[38;5;241;43m.\u001b[39;49m\u001b[43mis_complex\u001b[49m\u001b[43m(\u001b[49m\u001b[43m)\u001b[49m\u001b[43m \u001b[49m\u001b[38;5;28;43;01melse\u001b[39;49;00m\u001b[43m \u001b[49m\u001b[38;5;28;43;01mNone\u001b[39;49;00m\u001b[43m,\u001b[49m\u001b[43m \u001b[49m\u001b[43mnon_blocking\u001b[49m\u001b[43m)\u001b[49m\n",
+      "\u001b[0;31mOutOfMemoryError\u001b[0m: CUDA out of memory. Tried to allocate 2.00 MiB (GPU 0; 7.79 GiB total capacity; 6.47 GiB already allocated; 512.00 KiB free; 6.59 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF"
+     ]
+    }
+   ],
    "source": [
-    "train_dataloader = DataLoader(training_data, batch_size, num_workers=4, shuffle=True, drop_last=True)\n",
+    "\n",
+    "\n",
+    "train_dataloader = DataLoader(training_data, batch_size, num_workers=number_of_workers, shuffle=True, drop_last=True)\n",
     "discriminator = Discriminator().to(device)\n",
     "generator = Generator().to(device)\n",
     "d_optimizer = torch.optim.Adam(discriminator.parameters(), d_lr, [beta1, beta2])\n",
     "g_optimizer = torch.optim.Adam(generator.parameters(), g_lr, [beta1, beta2])\n",
+    "#TOCHECK why is the reasong to choose 64?\n",
```

Comment:
Hi @seansunn , can you please help with this one?

Reply:
Hi @mxochicale , 64 here is the parameter for feature of the FID. According to the original author [here](https://github.com/bioinf-jku/TTUR#fr%C3%A9chet-inception-distance-fid):

_IMPORTANT: The **number of samples** to calculate the Gaussian statistics (mean and covariance) **should be greater than the dimension of the coding layer**, here 2048 for the Inception pool 3 layer. Otherwise the covariance is not full rank resulting in complex numbers and nans by calculating the square root._

Since we only have about 100 samples for evaluation, we may need to set the feature size to 64. (The other options are: 192, 768, 2048, from the [torchmetrics' implementation of FID](https://torchmetrics.readthedocs.io/en/stable/image/frechet_inception_distance.html))

Reply:
This is great explanation, would you please add those lines in relevant part of the notebook. Thanks, --Miguel

Reply:
Done! I've added the explanation to both notebooks